intermediate_source/rpc_tutorial.rst

```diff
@@ -0,0 +1,555 @@
+Getting Started with Distributed RPC Framework
+=================================================
+**Author**: `Shen Li <https://mrshenli.github.io/>`_
+
+
+This tutorial uses two simple examples to demonstrate how to build distributed
+applications with the `torch.distributed.rpc` package. Source code of the two
+examples can be found in `PyTorch examples <https://github.com/pytorch/examples>`__
+
+Previous tutorials described `DistributedDataParallel <https://pytorch.org/docs/stable/_modules/torch/nn/parallel/distributed.html>`__
+which supports a specific training paradigm where the model is replicated across
+multiple processes and each process handles a split of the input data.
+Sometimes, you might run into scenarios that require different training
+paradigms:
+
+1) In reinforcement learning, it might be relatively expensive to acquire
+   training data from environments while the model itself can be quite small. In
+   this case, it might be useful to spawn multiple observers running in parallel
+   and share a single agent. In this case, the agent takes care of the training
+   locally, but the application would still need libraries to send and receive
+   data between observers and the trainer
+2) Your model might be too large to fit in GPUs on a single machine, and hence
+   would need a library to help split a model onto multiple machines. Or you
+   might be implementing a parameter server training framework, where model
```

Comment:
nit: Should we add a link to the parameter server paper for those not familiar? https://www.cs.cmu.edu/~muli/file/parameter_server_osdi14.pdf

Reply:
yes, will add, thanks!